components/src/core/drawer/drawer-nav-item.tsx

```diff
@@ -49,11 +50,19 @@ const makeStyles = (props: DrawerNavItemProps): any =>
             borderBottomRightRadius: props.navItem.activeItemBackgroundShape === 'square' ? 0 : 24,
             opacity: 0.9,
         },
+        expandIcon: {
+            display: 'flex',
+            height: 48,
+            width: 48,
+            marginRight: -12,
+            alignItems: 'center',
+            justifyContent: 'space-around',
+        },
```

Comment:
I think I've changed my mind about following the react styles. These are kind of gross. Can we just do:
```suggestion
        expandIcon: {
            display: 'flex',
            alignItems: 'center',
            marginLeft: 16,
        },
```